core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -270,6 +270,7 @@ coreos:
             {{end -}}
 
             [Service]
+            Type=simple
```

Comment:
etcd rollout stalls at the first etcd server in a new cluster when the etcdadm-check (etcd disaster recovery enabled) service is enabled.  This service, etcd-member, is type 'notify' by default - but etcd won't send the notify until the cluster is healthy (and so systemd reports etcd as starting-not started even though etcd is running ok).  Etcadm-check won't start until etcd-member has started and so no cfn-signal can be sent to start the subsequent etcd servers.  This means that your roll out gets stuck at the first etcd server.  By changing the type of etcd-member to 'simple' we allow etcdadm-check start and the cloud-formation signal can go out.  There is no side effect from changing the type so that as soon as etcd has been started everything can continue.

Reply:
@davidmccormick Good point!

But I take this change to actually result in a side-effect of we no longer postpone the rolling update of etcd nodes in case etcd-member failed to join the existing cluster. I think this is better explained in https://github.com/kubernetes-incubator/kube-aws/issues/1206#issuecomment-378857233.

Not saying this change is no good. However, at least, this chance should be discussed and made separately!
That being said, would you mind reverting this?